static/src/js/containers/SavedProjectsContainer/SavedProjectsContainer.jsx

```diff
@@ -1,66 +1,125 @@
 import { withRouter } from 'react-router-dom'
 
 import actions from '../../actions'
+
 import { SavedProjects } from '../../components/SavedProjects/SavedProjects'
+import ProjectRequest from '../../util/request/projectRequest'
+import { addToast } from '../../util/addToast'
+import { getEarthdataEnvironment } from '../../selectors/earthdataEnvironment'
 
 export const mapStateToProps = (state) => ({
-  savedProjects: state.savedProjects.projects,
-  savedProjectsIsLoading: state.savedProjects.isLoading,
-  savedProjectsIsLoaded: state.savedProjects.isLoaded
+  authToken: state.authToken,
+  earthdataEnvironment: getEarthdataEnvironment(state)
 })
 
 export const mapDispatchToProps = (dispatch) => ({
-  onDeleteSavedProject: (projectId) => dispatch(actions.deleteSavedProject(projectId)),
-  onFetchSavedProjects: () => dispatch(actions.fetchSavedProjects()),
-  onChangePath: (path) => dispatch(actions.changePath(path))
+  onChangePath: (path) => dispatch(actions.changePath(path)),
+  dispatchHandleError: (errorConfig) => dispatch(actions.handleError(errorConfig))
 })
 
-export class SavedProjectsContainer extends Component {
-  componentDidMount() {
-    const {
-      onFetchSavedProjects
-    } = this.props
-
-    onFetchSavedProjects()
-  }
-
-  render() {
-    const {
-      savedProjects,
-      savedProjectsIsLoading,
-      savedProjectsIsLoaded,
-      onChangePath,
-      onDeleteSavedProject
-    } = this.props
-
-    return (
-      <SavedProjects
-        savedProjects={savedProjects}
-        savedProjectsIsLoading={savedProjectsIsLoading}
-        savedProjectsIsLoaded={savedProjectsIsLoaded}
-        onChangePath={onChangePath}
-        onDeleteSavedProject={onDeleteSavedProject}
-      />
-    )
-  }
+export const SavedProjectsContainer = ({
+  onChangePath,
+  authToken,
+  earthdataEnvironment,
+  dispatchHandleError
+}) => {
+  const [projects, setProjects] = useState([])
+  const [isLoading, setIsLoading] = useState(false)
+  const [isLoaded, setIsLoaded] = useState(false)
+
+  const fetchProjects = useCallback(async () => {
+    setIsLoading(true)
+    setIsLoaded(false)
+
+    if (!authToken) {
+      setIsLoading(false)
+      setProjects([])
+
+      return
+    }
+
+    try {
+      const requestObject = new ProjectRequest(authToken, earthdataEnvironment)
+      const response = await requestObject.all()
+      setProjects(response.data)
+      setIsLoaded(true)
+    } catch (error) {
+      dispatchHandleError({
+        error,
+        action: 'fetchProjects',
+        resource: 'saved projects',
+        verb: 'fetching',
+        notificationType: 'banner'
+      })
+
+      addToast('Error fetching saved projects. Please try again.', {
+        appearance: 'error',
+        autoDismiss: true
+      })
+    } finally {
+      setIsLoading(false)
+    }
+  }, [authToken, earthdataEnvironment, dispatchHandleError])
+
+  const handleDeleteSavedProject = useCallback(async (projectId) => {
```

Comment:
This delete logic was done differently in this PR than in the DownloadHistory PR, but I think this version is much better. 

In this version the container handles the confirm, and the display component (SavedProjects) only has to call a function passed in as a prop.

In DownloadHistory the display component is doing the confirm before calling the prop function, but there are two useCallback functions that seem repetitive.

In order to make sure our code is consistent, can you refactor the DownloadHistory delete logic to mirror what you've done in this PR?